70.app-update-module/docs.md

```diff
@@ -0,0 +1,26 @@
+---
+title: app-update-module
+taxonomy:
+    category: docs
+shortcode-core:
+    active: false
+github: false
+---
+
+## app-update-module 1.0.0
```

Comment:
For the very first release, we usually omit changelogs. See for example [mender-configure](https://docs.mender.io/release-information/release-notes-changelog/mender-configure-module#changelogs-7).